README.md

```diff
@@ -63,6 +63,7 @@ specific dependencies please visit the single package's documentation:
 | v1.0.0                              |                    | :white_check_mark: |                    |
 | v1.1.0                              | :white_check_mark: | :white_check_mark: | :x:                |
 | v1.2.0                              | :white_check_mark: | :white_check_mark: | :x:                |
+| v1.3.0                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |
```

Comment:
just to be clear, which are the reasons to bump minor and not patch version?

Reply:
probably these reasons should be clearly stated in the changelog, e.g. dividing between breaking/non breaking changes.

Reply:
As i wrote in the description of the PR:

> I increased a minor version because we include new features (goldpinger).

Reply:
ops, sorry, you are right, LGTM